src/App.js

```diff
@@ -1,13 +1,119 @@
 import React from 'react';
 import './App.css';
+import Product from "./components/Products";
+import Button from "./components/Button";
+import Tile from "./components/Tile";
+
+import bag_1 from "./assets/bag_1.png";
+import bag_2 from "./assets/bag_2.png";
+import bag_3 from "./assets/bag_3.png";
+import bag_4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import story from "./assets/our_story.png";
+
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
 
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+
+            <nav>
+
+                <Button
+                    text="To the collection"
+                    disable={false}
+                />
+                <Button
+                    text="Shop all bags"
+                    disable={false}
+
+                />
+                <Button
+
+                    text="Pre-Orders"
+                    disable={true}
+
+                />
+
+            </nav>
+
+            <main>
+                <Product
+                    tag="Best seller"
+                    image={bag_1}
+                    imageAlt="bag_1"
+                    description="The handy bag"
+                    price="€400,-"
```

Comment:
Bij het invullen van een prijs kan je eventueel ook in het Product.js component bestand al een '€' en ',-' erin zetten. Dus bijvoorbeeld <p>€{price},-</p> - Dan hoef je in het App.js bij price alleen nog het getal in te vullen